### ISSUE TITLE
Contrib documentation - lacking notes on how to compile assets to manifests

### ISSUE THREAD

**Issue by gdmello** [coreos/prometheus-operator#487]
**What did you do?**
New to `prometheus` and the `prometheus-operator`, and trying to understand how to use this operator. In particular was looking for a way to add to the existing alert rules in prometheus for my kubernetes cluster.

**What did you expect to see?**
Documentation describing how to push rule modifications/ additions in the `contrib/kube-prometheus/assets` to the `contrib/kube-prometheus/manifests` dir.

**What did you see instead? Under which circumstances?**
No such documentation.

**Environment**
My k8s cluster on AWS.

* Kubernetes version information:

```
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.2", GitCommit:"477efc3cbe6a7effca06bd1452fa356e2201e1ee", GitTreeState:"clean", BuildDate:"2017-04-19T20:33:11Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
```

* Kubernetes cluster kind: 

```
kube-spray
```
I can contribute the missing docs in a PR, if the authors agree to this Issue.

**Comment by brancz** [comment]
Yes this is definitely something we want to add at some point, currently we're still missing some documentation for the core functionality of the Prometheus Operator that will probably be added first, then we can go on to docs about things like this.

As a hint, from the `contribu/kube-prometheus` directory there is a `hack/scripts` directory that contains she `generate-manifests.sh` script that will generate all the k8s manifests from the `assets/` directory.

If you want to we would highly appreciate if you could add a short doc on this workflow.

**Comment by gdmello** [comment]
@brancz - yes I certainly could! Thanks for your response.

**Comment by brancz** [comment]
Awesome! Thanks a lot, highly appreciated!

**Comment by brancz** [comment]
As more people are showing interest in this, I'll take this over. Is that ok with you @gdmello ?

**Comment by brancz** [comment]
Closed by #505

**Comment by gdmello** [comment]
Sure! thanks for addressing it @brancz!
